lib/GcInfo/GcInfo.cpp

```diff
@@ -57,6 +58,15 @@ bool GcInfo::isGcAggregate(const Type *AggType) {
   return false;
 }
 
+bool GcInfo::isGcAllocation(const llvm::Value *Value) {
+  const AllocaInst *Alloca = dyn_cast<const AllocaInst>(Value);
+  if ((Alloca != nullptr) && GcInfo::isGcType(Alloca->getAllocatedType())) {
+    return true;
+  }
+
+  return false;
+}
+
```

Comment:
Seems like the latter bit of code could simply be

`return (Alloca != nullptr) && GcInfo::isGCType(...);`